plugins/org.jkiss.dbeaver.ext.oracle/src/org/jkiss/dbeaver/ext/oracle/model/OracleDataType.java

```diff
@@ -103,7 +103,7 @@ private TypeDesc(DBPDataKind dataKind, int valueType, int precision, int minScal
         PREDEFINED_TYPES.put("POINTER", new TypeDesc(DBPDataKind.UNKNOWN, Types.OTHER, 0, 0, 0));
         PREDEFINED_TYPES.put("REAL", new TypeDesc(DBPDataKind.NUMERIC, Types.REAL, 38, 127, -84));
         PREDEFINED_TYPES.put("REF", new TypeDesc(DBPDataKind.REFERENCE, Types.OTHER, 0, 0, 0));
-        PREDEFINED_TYPES.put("SIGNED BINARY INTEGER", new TypeDesc(DBPDataKind.NUMERIC, Types.INTEGER, 38, 127, -84));
+        //PREDEFINED_TYPES.put("SIGNED BINARY INTEGER", new TypeDesc(DBPDataKind.NUMERIC, Types.INTEGER, 38, 127, -84)); can not be created as a data type
```

Comment:
Shouldn't we left it for older versions?

Reply:
This is tricky. Because I didn't find any documentation about this type. 
So "old" here - this is "probably old". It may not have worked in previous versions, too.

Reply:
There's also `UNSIGNED BINARY INTEGER`. Should we remove it too?

I was able to find some mentions of `BINARY_INTEGER`: https://www.orafaq.com/wiki/BINARY_INTEGER, so maybe we should remove its `SIGNED`/`UNSIGNED` variants and just leave `BINARY_INTEGER`?

Reply:
And you will get SQL Error [902] [42000]: ORA-00902: invalid datatype with BINARY_INTEGER